asciidoc-java/src/main/java/io/yupiik/asciidoc/parser/Parser.java

```diff
@@ -1452,10 +1464,7 @@ private Map<String, String> readAttributes(final Reader reader, final ContentRes
                             ofNullable(reader.nextLine()).map(String::strip).map(it -> ' ' + it).orElse("");
                 }
                 attributes.put(matcher.group("name"), value);
-            } else if (attributes.isEmpty()) {
-                reader.rewind();
-                break;
-            } else {
+            } else if (MACRO.matcher(line).matches()) {
```

Comment:
not sure the regex is true since the macro doesn't have to start the line and if we go with a regex i'd like to have a quick test before (line.contains("::") for ex) since most of the time we'll not match it

Reply:
That makes sense for the check, we could have `isMacro(line)`. The regex is generated by AI, we need to test it and improve it for sure.